src/app/content/practiceQuestions/types.ts

```diff
@@ -30,3 +30,8 @@ export interface PracticeQuestion {
 }
 
 export type PracticeQuestions = PracticeQuestion[];
+
+export type PracticeQuestionsLocationFilters = Map<
```

Comment:
this shouldn't be used now right?

Reply:
I'm using this in `getPracticeQuestionsLocationFilters`
The difference between this and `LocationFilters` is that a `section` has to be `LinkedArchiveTree` and it has to have `children`

Reply:
is that necessary? or can you just use `LocationFilters` everywhere?

Reply:
I could use `LocationFilters` but then I would have to assert that `section` is `LinkedArchiveTree` and `children` is always present in a few places. I probably could create a separate type for this kind of filters